### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by blehhtech**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by blehhtech — COMMENTED** [review]
It's nice

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (blehhtech) [inline index.html:49]: blue balloons on top a tree

Comment (od41) [reply index.html:49]: hii

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (blehhtech) [inline index.html]:
```suggestion
<h4 hidden>weldone try again</h4>
```

**Pull request merged**